feedback/static/html/studio_view.html

```diff
@@ -0,0 +1,111 @@
+        </label>
+      </div>
+      <span class="tip setting-help">This is shown as grayed out text before the student has answered.</span>
+    </li>
+
+    <li class="field comp-setting-entry is-set">
+      <div class="wrapper-comp-setting">
+        <label class="label setting-label">Likert icon set
+	<select name="icon_set">
+	  <option value="face">Faces (happy to sad)</option>
+	  <option value="midface">Faces (sad to happy to sad)</option>
+	  <option value="num">Numeric</option></select>
+	</select>
+	</label>
+      </div>
+      <span class="tip setting-help">We can either show happy/sad faces, or numbers 1-5.</span>
+    </li>
+
+    <li class="field comp-setting-entry is-set">
+      <div class="wrapper-comp-setting">
+        <label class="label setting-label">Likert prompt
+          <input class="input setting-input" name="likert" id="likert" value="{likert}" type="text" />
+        </label>
+      </div>
+      <span class="tip setting-help">Example: Please rate your overall experience with this section.</span>
+    </li>
+
+    <li class="field comp-setting-entry is-set">
+      <div class="wrapper-comp-setting">
+        <label class="label setting-label">Likert option 1
+          <input class="input setting-input" name="likert0" id="likert0" value="{likert0}" type="text" />
+        </label>
+      </div>
+      <span class="tip setting-help">Example: Excellent!</span>
+    </li>
+
+    <li class="field comp-setting-entry is-set">
+      <div class="wrapper-comp-setting">
+        <label class="label setting-label">Likert option 2
+          <input class="input setting-input" name="likert1" id="likert1" value="{likert1}" type="text" />
+        </label>
+      </div>
+      <span class="tip setting-help">Example: Good</span>
+    </li>
+
+    <li class="field comp-setting-entry is-set">
+      <div class="wrapper-comp-setting">
+        <label class="label setting-label">Likert option 3
+          <input class="input setting-input" name="likert2" id="likert2" value="{likert2}" type="text" />
+      </div>
+      <span class="tip setting-help">Example: Average</span>
+    </li>
+
+    <li class="field comp-setting-entry is-set">
+      <div class="wrapper-comp-setting">
+        <label class="label setting-label">Likert option 4</label>
+          <input class="input setting-input" name="likert3" id="likert3" value="{likert3}" type="text" />
+        </label>
+      </div>
+      <span class="tip setting-help">Example: Fair</span>
+    </li>
+
+    <li class="field comp-setting-entry is-set">
+      <div class="wrapper-comp-setting">
+        <label class="label setting-label">Likert option 5</label>
+          <input class="input setting-input" name="likert4" id="likert4" value="{likert4}" type="text" />
+        </label>
+      </div>
+      <span class="tip setting-help">Example: Poor</span>
+    </li>
+  </ul>
+
+  <div class="xblock-actions">
+    <ul>
```

Comment:
I'm not sure this actually qualifies as a "list" but acknowledge that its probably a studio convention.  We should fix that at some point.